src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java

```diff
@@ -970,7 +973,7 @@ protected boolean isCategoryEnabled(@NonNull SCMHeadCategory category) {
     public void afterSave() {
         SCMSourceOwner owner = getOwner();
         if (owner != null) {
-            GitHubWebHook.get().registerHookFor(owner);
+            // TODO Post JENKINS-39533 GitHubWebHook.get().registerHookFor(owner);
```

Comment:
39553 and I actually need those changes or events are not received so I backed out these uncomments